.github/workflows/build.yml

```
@@ -28,22 +28,22 @@ jobs:
-  #         cache: "yarn"
-  #       env:
-  #         SKIP_YARN_COREPACK_CHECK: "1"
-  #     - name: Enable corepack
-  #       run: corepack enable
-  #     - name: Install dependencies
-  #       run: yarn
-  #     - name: Test client
-  #       env:
-  #         OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
-  #       run: yarn test:client
+    test:
```

actionlint (1.7.3)

unexpected key "test" for "job" section. expected one of "concurrency", "container", "continue-on-error", "defaults", "env", "environment", "if", "name", "needs", "outputs", "permissions", "runs-on", "secrets", "services", "steps", "strategy", "timeout-minutes", "uses", "with"

(syntax-check)